### DISCUSSION TITLE
Rancher failed to install longhorn

### DISCUSSION THREAD

**Discussion by tqq1994516** [longhorn/longhorn#5347] [Q&A]
When installing longhorn through ranhcer in the Windows cluster, the longhorn-admission-webhook pod has been waiting for the longhorn-conversion-webhook, but the longhorn-conversion-webhook pod has been in the running state, causing the subsequent pods to be waiting
![image](https://user-images.githubusercontent.com/61398122/216597619-ecd0cc75-393c-4264-91bd-ef3d56647c7b.png)
There is no useful information in  longhorn-conversion-webhook pod log
![image](https://user-images.githubusercontent.com/61398122/216598433-6232363f-5af7-4904-b35f-bcdc95d1bb35.png)
The Windows cluster option has been selected during installation
![image](https://user-images.githubusercontent.com/61398122/216598763-a964c5c0-7f16-42c9-8f5d-07069ecb2d4a.png)

**Comment by weizhe0422** [comment]
If this situation continues to happen, could you please also provide the log of the admission webhook pod?
